Automattic/jetpack
# AI Assistant: remove Experimental label

Fixes #35795 

## Proposed changes:
This PR removes the "Experimental" label on the block name

### Other information:


## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
Go to the editor and type `/ai` so the inserter shows the block on the dropdown. See that the block name is just "AI Assistant" (no "Experimental" label present).

Check the on the block, pre and post publish sidebar panels, there should be no mention to "Experimental"
